test/_functions/common/event-recorder/golang/event_recorder.go

```diff
@@ -58,10 +57,7 @@ func Handler(context *nuclio.Context, event nuclio.Event) (interface{}, error) {
 	}
 
 	// Read log file
-	data, err := ioutil.ReadFile(eventsLogFilePath)
-	if err != nil && !os.IsNotExist(err) {
-		return nil, fmt.Errorf("failed to read log file: %w", err)
-	}
+	data, _ := os.ReadFile(eventsLogFilePath)
```

Comment:
If an error does occur, does it return an empty string?

Reply:
@TomerShor empty byte slice